compiler/noirc_frontend/src/hir/comptime/errors.rs

```diff
@@ -96,12 +99,22 @@ impl InterpreterError {
             | InterpreterError::UnquoteFoundDuringEvaluation { location, .. }
             | InterpreterError::Unimplemented { location, .. }
             | InterpreterError::BreakNotInLoop { location, .. }
-            | InterpreterError::ContinueNotInLoop { location, .. } => *location,
+            | InterpreterError::ContinueNotInLoop { location, .. }
+            | InterpreterError::DebugEvaluateComptime { location, .. } => *location,
             InterpreterError::Break | InterpreterError::Continue => {
                 panic!("Tried to get the location of Break/Continue error!")
             }
         }
     }
+
+    pub(crate) fn debug_evaluate_comptime(expr: impl Display, location: Location) -> Self {
+        let diagnostic = CustomDiagnostic::simple_debug(
+            "`comptime` expression ran:".to_string(),
+            format!("After evaluation:\n{}", expr),
```

Comment:
```suggestion
            format!("After evaluation: {}", expr),
```
We should add the check you mentioned here for single-line expressions. Otherwise it looks like a bug when you see another number in the margin for code which evaluated to an integer. I imagine most users wouldn't see it at all.

Reply:
Ah yeah, I disregarded the result in my example due to the fact that it was in the margin and it looked exactly like a line number.